src/HotChocolate/Data/src/Data/Filters/Types/ListFilterInput`1.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace HotChocolate.Data.Filters
 {
-    public class ListFilterInput<T> : FilterInputType, IListFilterInputType
+    public class ListFilterInput<T> : FilterInputType, IListFilterInput
```

Comment:
if we have more than one implementation break this 

: FilterInputType
, IListFilterInput